### PR TITLE
Feature/regions

### PR DESCRIPTION
**Pull request by jlconlin**

### CODE REVIEW

File: src/ACEtk/interpretation/Region/src/makeRegions.hpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// static int makeRegions( ENDFInterpolationParameters eip,
```
Comment (apmccartney) [inline src/ACEtk/interpretation/Region/src/makeRegions.hpp]: file can be deleted

**Comment by apmccartney** [comment]
While I like the Region class you've written, this isn't at all what we discussed this morning

File: src/ACEtk/interpretation/Region/src/ctor.hpp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Region( int INT, xRange&& x, yRange&& y )
+try:
+  interpolationScheme_( verifyInterpolationScheme( INT ) ),
+  x_( x ), y_( y )
```
Comment (apmccartney) [inline src/ACEtk/interpretation/Region/src/ctor.hpp]: You're taking rvalue references to x and y. Don't copy. Move instead.

Comment (jlconlin) [reply src/ACEtk/interpretation/Region/src/ctor.hpp]: Yeah, this didn't feel right, but I wasn't quite sure how to do this.

File: src/ACEtk/interpretation/Region/src/makeRegions.hpp (hunk 1 above)
Comment (jlconlin) [reply src/ACEtk/interpretation/Region/src/makeRegions.hpp]: Placeholder, as discussed.

File: src/ACEtk/interpretation/ENDFInterpolationParameters.hpp [hunk 3]
```diff
@@ -5,6 +5,11 @@ class ENDFInterpolationParameters {
 public:
 #include "ACEtk/interpretation/ENDFInterpolationParameters/src/ctor.hpp"
 
+  auto operator[]( int index ) const {
```
[the remark this answers is not in the document]
Comment (jlconlin) [reply src/ACEtk/interpretation/ENDFInterpolationParameters.hpp]: Debatable :-D

**Pull request merged**